src/translator/vocabs.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+namespace marian {
+namespace bergamot {
+
+class VocabsGenerator {
+public:
+  VocabsGenerator(Ptr<Options> options): options_(options){}
```

Comment:
Should be aware of only Filenames / AlignedMemory and not aware of `Ptr<Options>` - which are two modes of construction.